### ISSUE TITLE
Algolia Not Loading Assets Into <head>

### ISSUE THREAD

**Issue by nickpiro** [algolia/algoliasearch-magento#1015]
Installed Algolia module on 1.9.1.1 and it seems the layout xml file is not loading assets (.js .css) to the `<head>`. The search box switched to the Algolia search box but no .js or .css files were added. Making me assume the whole plugin would not work! Which it doesn't. Any suggestions or similar situations?

**Comment by JanPetr** [comment]
Hello @nickpiro,

to be honest it's the first time I see this issue. Do you have merging of static files enabled?
Cannot it be caused by cache? Cat you try to disable it?

**Comment by nickpiro** [comment]
Hey @JanPetr, I ended up figuring this out! It was due to our server running an OLD version of PHP. I just took over the website, which was running 1.9.1.1 (so I assumed PHP was up to date). I ended up seeing that it was breaking at an array, so when I delved further and realized that it was due to PHP. Once I updated it everything worked fine.

I'm having a new issue on our new M2 install though! Which I was going to comment about today. 

Thanks!
